<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgYJZH</class>
 <widget class="QDialog" name="DlgYJZH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>一键转化</string>
  </property>
  <widget class="QPushButton" name="btnOk">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>30</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>图纸选择</string>
   </property>
   <widget class="QRadioButton" name="rdBtnImp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>导入图纸</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdBtnCur">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>使用当前图纸</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>DlgYJZH</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DlgYJZH</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
